gql-queries/vol_qry.r

```diff
@@ -0,0 +1,36 @@
+#install.packages("glue")
+library(glue)
+
+# GQL for volumes, task 4b
+vol_qry <- function(id, from, to) {
+  query <- glue('{
+    trafficData(trafficRegistrationPointId: <<id>>) {
+      volume {
+        byHour(from: <<from>>, to: <<to>>) {
+          edges {
+            node {
+              from
+              to
+              total {
+                volumeNumbers {
+                  volume
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }', .open = "<<", .close = ">>")
+  cat(query)
+}
```

Comment:
The error you are facing in the last task is probably due to this function not being completely correct. If you look at the example API input in the task, you see that the Id, from and to are inside quotation marks " ". This results in wrong data from the GQL() function in the iterations.r script.